distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Thread.enso

```diff
@@ -3,7 +3,8 @@
 
 import project.Any.Any
 
-## ADVANCED
+## PRIVATE
+   ADVANCED
 
    Executes an action with a handler for the executing thread being
    interrupted.
```

Comment:
small nit: I assume the blank line may be removed, seeing that you did remove these in some other places.